openapi.yaml

```diff
@@ -49,156 +95,494 @@ paths:
-  PaginationLinks:
+  OwnerResultPrivate:
     properties:
-      first:
-        type: string
-        format: url
-        description: The first page of data
-      last:
-        type: string
-        format: url
-        description: The last page of data
-      prev:
-        type: string
-        format: url
-        description: The previous page of data
-      next:
+      links:
+        $ref: '#/definitions/SelfLink'
+      data:
+        $ref: '#/definitions/OwnerResourcePrivate'
+  OwnerResultPublic:
+    properties:
+      links:
+        $ref: '#/definitions/SelfLink'
+      data:
+        $ref: '#/definitions/OwnerResourcePublic'
+  OwnersResult:
+    properties:
+      links:
+        $ref: '#/definitions/SelfLink'
+      data:
+        type: array
+        items:
+          $ref: '#/definitions/OwnerResourcePublic'
+  OwnerResourcePublic:
+    properties:
+      id:
         type: string
-        format: url
-        description: The next page of data
-  PetResource:
+        example: 0001
+        description: A unique ID of a park owner
+      attributes:
+        properties:
+          ownerName:
+            type: string
+            description: Name of park owner / organization
+            example: City of Corvallis
+  OwnerResourcePrivate:
     properties:
       id:
         type: string
-        description: A unique ID of pet
-      type:
+        example: 0001
+        description: A unique ID of a park owner
+      attributes:
+        properties:
+          ownerName:
+            type: string
+            description: Name of park owner / organization
+            example: City of Corvallis
+          ownerPassword:
+            type: string
+            description: Password
+            example: password1234
+  ParkResource:
+    properties:
+      id:
         type: string
-        enum: [pet]
-      links:
-        $ref: '#/definitions/SelfLink'
+        example: 0001
+        description: A unique ID of park
       attributes:
         properties:
           name:
             type: string
-            description: Name of pet
-            example: Hedwig
+            description: Name of Park
+            example: Avery Park
           owner:
+            type: object
+            properties:
+              ownerName:
+                type: string
+                example: City of Corvallis
+              ownerId:
+                type: string
+                example: 1111
+          location:
+            type: object
+            properties:
+              streetAddress:
+                type: string
+                description: Park address
+                example: 1200 SW Avery Park Dr.
+              city:
+                type: string
+                description: city where the park is located
+                example: Corvallis
+              state:
+                type: string
+                description: Abbreviation of state where park is located
+                example: OR
```

Comment:
For this string and any others that you want to restrict to a certain format, you can use the `pattern` field with a regular expression